test/components/gridlinesTests.ts

```diff
@@ -1,62 +1,62 @@
 ///<reference path="../testReference.ts" />
 
 describe("Gridlines", () => {
-
-  it("Scale ranges are set to the Gridlines dimensions when layout is computed", () => {
-    let svg = TestMethods.generateSVG(640, 480);
-    let xScale = new Plottable.Scales.Linear();
+  const SVG_WIDTH = 640;
+  const SVG_HEIGHT = 480;
+  let svg: d3.Selection<void>;
+  let xScale: Plottable.Scales.Linear;
+  let yScale: Plottable.Scales.Linear;
+  let gridlines: Plottable.Components.Gridlines;
+
+  beforeEach(() => {
+    svg = TestMethods.generateSVG(640, 480);
+    xScale = new Plottable.Scales.Linear();
     xScale.domain([0, 10]);
 
-    let yScale = new Plottable.Scales.Linear();
+    yScale = new Plottable.Scales.Linear();
     yScale.domain([0, 10]);
+    gridlines = new Plottable.Components.Gridlines(xScale, yScale);
+  });
 
-    let gridlines = new Plottable.Components.Gridlines(xScale, yScale);
+  it("sets ranges of scales to the Gridlines dimensions when layout is computed", () => {
     gridlines.renderTo(svg);
 
-    assert.deepEqual(xScale.range(), [0, 640], "x scale range extends to the width of the svg");
-    assert.deepEqual(yScale.range(), [480, 0], "y scale range extends to the height of the svg");
+    assert.deepEqual(xScale.range(), [0, SVG_WIDTH], "x scale range extends to the width of the svg");
+    assert.deepEqual(yScale.range(), [SVG_HEIGHT, 0], "y scale range extends to the height of the svg");
```

Comment:
If we get rid of `SVG_WIDTH` and `SVG_HEIGHT`, probably use `numAttr(svg, ...)` here.